src/cromshell/status/command.py

```diff
@@ -57,11 +58,18 @@ def main(config, workflow_id):
     elif workflow_status == "Running":
         # Status claims this workflow is running fine, but we need to check to see
         # if there are any failed sub-processes.
-        # To do this, we use the `execution-status-count` logic with some filtering:
-        # TODO : Use this as a template for the Metadata subcommand
-        # Get execution status count and filter the metadata down:
+        # To do this, we get the workflow metadata and search for any failures
+        formatted_metadata_parameter = metadata_command.format_metadata_params(
```

Comment:
Since this is being used in two different commands let me know if it should be relocated in the utility file, or if it is ok to leave it in the metadata command